src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/PhysicalFileProviderTests.cs

```diff
@@ -1479,5 +1530,31 @@ public void CreateFileWatcher_CreatesWatcherWithPollingAndActiveFlags()
                 Assert.True(fileWatcher.UseActivePolling);
             }
         }
+
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34580", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
+        public async Task CanDeleteWatchedDirectory(bool useActivePolling)
+        {
+            using (var root = new DisposableFileSystem())
+            using (var provider = new PhysicalFileProvider(root.RootPath))
+            {
+                var fileName = Path.GetRandomFileName();
+                var fileLocation = Path.Combine(root.RootPath, fileName);
```

Comment:
I don't see a usage of `fileLocation`.

Reply:
Good catch, I had that in a previous version, but had to remove it due to #44484.  Will clean it up.